packages/cubejs-client-core/index.d.ts

```diff
@@ -754,7 +759,7 @@ declare module '@cubejs-client/core' {
   export type DateRange = string | [string, string];
 
   export type TimeDimensionBase = {
-    dimension: string;
+    dimension: GeneratedTimeDimension;
```

Comment:
We'll need to handle this type somehow, as we use this `index.d.ts` for automatic docs generation but generated types will be different for each user.